src/go/rpk/pkg/cli/cmd/redpanda/start.go

```diff
@@ -143,25 +141,26 @@ func NewStartCommand(
 			// (POSIX standard)
 			UnknownFlags: true,
 		},
-		RunE: func(ccmd *cobra.Command, args []string) error {
+		RunE: func(cmd *cobra.Command, args []string) error {
```

Comment:
Good time to switch away from `RunE`?

Reply:
We can leave this for another PR; changing this here will be easy but will need a rewrite to our biggest unit test in RPK, which is the one for this command.